websites/.vuepress/config.js

```diff
@@ -1,4 +1,17 @@
+const fs = require('fs')
+const path = require('path')
+
+
 module.exports = {
+    generated() {
```

Comment:
That's for the redirects file, isn't it?

Reply:
yes, for now, we only have code to copy `_redirects` to `dist`. It's a vuepress `after build` hook i.e. if we need to anything once vuepress has done its job of `building` the site. This hook is only available for the `build` command but not for the `dev`

Reply:
Nice, can we move it a bit further down in the file?

Reply:
Alright, I'll move it to a separate file

Reply:
It's alright in the file - just thought about the placement within it :) Not a biggy anyways.